src/kaltura-player.js

```diff
@@ -677,6 +732,9 @@ class KalturaPlayer extends FakeEventTarget {
       this.play();
     }
   }
+  _onPlaybackStart(): void {
```

Comment:
I think it's better to set it on as inline listener if there isn't logic

Reply:
agree, will fix

Reply:
fixed